### PR TITLE
[Fixes #324] SITEURL is not being interpolated

### LINKED ISSUE #324: SITEURL is not being nterpolated
`SITEURL` must be interpolated with the `{hostname}` placeholder
https://github.com/GeoNode/geonode-project/blob/master/.env.sample#L43

### PR DESCRIPTION
**Pull request by marthamareal**
references #324

### CODE REVIEW

**Review by giohappy — APPROVED** [review]

File: .env.sample [hunk 1]
```diff
@@ -15,8 +15,6 @@ INVOKE_LOG_STDOUT=true
 
 DJANGO_SETTINGS_MODULE={{project_name}}.settings
 GEONODE_INSTANCE_NAME=geonode
-GEONODE_LB_HOST_IP=
-GEONODE_LB_PORT=
```
Comment (giohappy) [inline .env.sample:19]: why did you remove these @marthamareal?

Comment (marthamareal) [reply .env.sample:19]: Because they are duplicates. [here](https://github.com/GeoNode/geonode-project/blob/master/.env.sample#L67)

Comment (marthamareal) [reply .env.sample:19]: Because they are duplicates. [here](https://github.com/GeoNode/geonode-project/blob/master/.env.sample#L67)
